src/main/java/net/neoforged/neoforge/capabilities/RegisterCapabilitiesEvent.java

```diff
@@ -79,6 +80,34 @@ public boolean isBlockRegistered(BlockCapability<?, ?> capability, Block block)
         return capability.providers.containsKey(block);
     }
 
+    /**
+     * Makes a block capability proxyable.
+     *
+     * <p>This method should only be called by the mod that defines the capability,
+     * in the {@link EventPriority#HIGH} or {@link EventPriority#HIGHEST} phase.
+     *
+     * @throws IllegalStateException if the capability was marked to be non-proxyable
```

Comment:
I think this would be clearer as `@throws IllegalStateException if the capability was already marked as non-proxyable`